### PR TITLE
metaconfig: Update xinetd schema to include "port"

### PR DESCRIPTION
**Pull request by ned21**
port is a valid xinetd option according to `xinetd.conf(5)`.

Also remove default values for `cps` and `per_source`.

### CODE REVIEW

**Comment by ned21** [comment]
Please hold off on reviewing this, I have some more improvements coming.

I will also try to fix the panlint errors.

**Comment by ned21** [comment]
Ready for review now.

**Comment by jrha** [comment]
@hpcugentbot test this please

File: ncm-metaconfig/src/main/metaconfig/xinetd/pan/schema.pan [hunk 1]
```diff
@@ -15,16 +17,17 @@ type xinetd_options = {
     "type" ? xinetd_options_type[]
     "flags" ? xinetd_options_flags[]
     "only_from" ? xinetd_options_ips[]
-    "cps" : long[] = list(100, 2)
+    "cps" ? long[]
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/xinetd/pan/schema.pan]: Removing this default has broken a number of tests that were checking for it and now probably need updating.

```
not ok 43 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 44 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 14 before -1)
…
not ok 79 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 80 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 13 before -1)
…
not ok 117 - Found at least one match (total 0 matches) for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$)))
not ok 118 - Order ok for test idx 2 (pattern (?-xism:(?m:^\s{4}cps\s=\s100 2$))) (lastpos 13 before -1)
```

File: ncm-metaconfig/src/main/metaconfig/xinetd/pan/schema.pan [hunk 2]
```diff
@@ -15,16 +17,17 @@ type xinetd_options = {
...
     "user" : string = 'root'
     "server" ? string
-    "protocol" ? string with match(SELF,'^(udp|tcp)$') # actually, anything in /etc/protocols
+    "protocol" ? string with match(SELF, '^(udp|tcp)$') # actually, anything in /etc/protocols
     "server_args" ? string
     "group" ? string
-    "instances" ? long(0..) # if not defined, it's UNLIMITED
-    "per_source" : long = 11
+    "instances" ? string with match(SELF, '^(UNLIMITED|\d+)$')
+    "per_source" ? long
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/xinetd/pan/schema.pan]:
```
not ok 47 - Found at least one match (total 0 matches) for test idx 4 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$)))
not ok 48 - Order ok for test idx 4 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$))) (lastpos 16 before -1)
…
not ok 85 - Found at least one match (total 0 matches) for test idx 5 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$)))
not ok 86 - Order ok for test idx 5 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$))) (lastpos 32 before -1)
…
not ok 121 - Found at least one match (total 0 matches) for test idx 4 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$)))
not ok 122 - Order ok for test idx 4 (pattern (?-xism:(?m:^\s{4}per_source\s=\s11$))) (lastpos 15 before -1)
```

**Comment by ned21** [comment]
@jrha I had challenges running the tests locally but hopefully fixed now.

**Comment by jrha** [comment]
Certainly seem to be, many thanks!

**Pull request merged**